polaris_wyre/wyre/dtos.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from decimal import Decimal
+
+
+@dataclass
+class TransferData:
+    currency: str
+    amount: Decimal
+    destination: str
+
+    @classmethod
+    def create_transfer_data(cls, *args, **kwargs) -> "TransferData":
```

Comment:
Why is this method needed? Isn't its result the same as a simple construction like, for example:
`transfer_data = TransferData(currency="", amount=Decimal(), destination="")`

Reply:
I've fixed it on the next PR. Can I let this way for now?